workflow-examples/run_examples.sh

```diff
@@ -112,14 +100,7 @@ wait_project_start() {
 function execute_workflow() {
   params="$1"
   exitOnFailure="$2"
-  response=$(curl -X 'POST' -s \
-    "${TARGET_URL}/api/v1/workflows" \
-    -H 'accept: */*' \
-    -H 'Authorization: Basic dGVzdDp0ZXN0' \
-    -H "X-XSRF-TOKEN: ${TOKEN}" \
-    -b $COOKIEFP \
-    -H 'Content-Type: application/json' \
-    -d "$params")
+  response=$(parodos_v1_curl workflows -d "$params")
```

Comment:
Missing `-X 'POST'` here

Reply:
-d implies POST though. Infact I guess -X GET is also redundant most of the time here